charts/gatekeeper/README.md

```diff
@@ -104,7 +104,7 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 
 This Helm chart is autogenerated from the Gatekeeper static manifest. The
 generator code lives under `cmd/build/helmify`. To make modifications to this
-template, please edit `kustomization.yaml` and `replacements.go` under that
-directory and then run `make manifests`. Your changes will show up in the
-`manifest_staging` directory and will be promoted to the root `charts` directory
-the next time a Gatekeeper release is cut.
+template, please edit `kustomization.yaml`, `kustomize-for-helm.yaml` and
```

Comment:
can you make this change in the readme in `manifest_staging` please?

Reply:
done.